### ISSUE TITLE
autoload error

### ISSUE THREAD

**Issue by mat-cab** [cowai/camerarawpreviews#15]
Hi,

I tried installing on a brand new nextcloud server your app, and I can activate it without problem.
However, when accessing the gallery, I have no previews (only the "ongoing ring").

I find in the apache error log the following messages:
`[Sun Apr 08 13:29:53.418239 2018] [:error] [pid 1004] [client 0.0.0.0:0000] PHP Fatal error:  require(): Failed opening required '/var/www/html/nextcloud/apps/camerarawpreviews/lib/../vendor/autoload.php' (include_path='/var/www/html/nextcloud/3rdparty/pear/console_getopt:/var/www/html/nextcloud/3rdparty/pear/pear_exception:/var/www/html/nextcloud/3rdparty/pear/pear-core-minimal/src:/var/www/html/nextcloud/3rdparty/pear/archive_tar:/var/www/html/nextcloud/apps') in /var/www/html/nextcloud/apps/camerarawpreviews/lib/RawPreview.php on line 3
`
That autoload.php page is not existing in your project.
I see that in other "official" apps (gallery for example), this file does exist.

Maybe it is necessary to include another file ? Or include it directly in your project ? Or specify a particular include_path ?

PS: I also think there is a mistake in your README, there should be no escape char for \ in here:
 `'enabledPreviewProviders' => array(
  'OCA\\CameraRawPreviews\\RawPreview',
  'OCA\\CameraRawPreviews\\IndesignPreview' // for indesign files (.indd)
),
`

**Comment by cowai** [comment]
Hi,
Which PHP version do you have? As for the autoload.php it is part of the tar.gz appstore package. I have also tested the install on a new nextcloud 13 install and that worked.
How did you install it? If you are using git, you need to build it (run make). I recommend just install it in the nextcloud appstore.

About the second bug, I am not sure we even need them in your config.

**Comment by cowai** [comment]
Mentioned easier install method in the README.md https://github.com/cowai/camerarawpreviews/commit/a0d06505653821e022fce6ad3f30ba53a58be031

**Comment by mat-cab** [comment]
Yeah I can confirm that the app works as expected using the tarball from the app store.

Concerning the config, I left the mentionned lines in the config.php, therefore I guess that it works OK with them!

**Comment by mat-cab** [comment]
Since we are also discussing unimportant issues, I shall also add that the download of the git repository (in zip form) is detected by Avast antivirus as infected by ELF:Bew-L [Cryp]...

**Comment by cowai** [comment]
That is probably from the embedded Perl binary. I will try to make that
optional somehow.

**Comment by cowai** [comment]
Most distros have perl included, but wonderfall/nextcloud docker image does not and that is why I am including it. But I will try to include it from source.
The files is also not documented now. But its coming from http://software.schmorp.de/pkg/App-Staticperl.html
